source/in-use-encryption-tutorial.txt

```diff
@@ -0,0 +1,155 @@
+.. _encryption-tutorial:
+
+==========================
+In-Use Encryption Tutorial
+==========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+In-Use Encryption allows you to connect to your deployments using :v6.0:`Queryable 
+Encryption </core/queryable-encryption/>`. This connection method allows you to 
+encrypt a subset of fields in your collections. 
+
+Overview
+--------
+
+This guide shows you how to connect to your deployment and collections using 
+Queryable Encryption. 
+
+This guide uses the `air_airlines.json <https://raw.githubusercontent.com/mongodb/docs-assets/compass/air_airlines.json>`__
+data set in the guided examples. The guide covers the process of importing 
+your data set. 
+
+Requirements and Limitations
+----------------------------
+
+- In-Use Encryption is an Enterprise/Atlas only feature. 
+
+- You need a replica set to use this connection option. Your replica set can be 
+  a single node or larger.
+
+- You need to connect to your deployment on Compass using In-Use Encryption. For 
+  more information on how to connect your deployment, see :ref:`In-Use Encryption 
```

Comment:
[nit]

One minor bit here -- I'd suggest "how to connect to your deployment" on line 38.

Reply:
Done.